src/databricks/labs/remorph/reconcile/execute.py

```diff
@@ -297,3 +304,40 @@ def _compute_threshold_comparison(self, table_conf: Table, src_schema: list[Sche
             threshold_df = mismatched_df.limit(_SAMPLE_ROWS)
 
         return ThresholdOutput(threshold_df=threshold_df, threshold_mismatch_count=mismatched_count)
+
+
+def get_schema(
+    source: DataSource, target: DataSource, table_conf: Table, database_config: DatabaseConfig
```

Comment:
```suggestion
    source: DataSource, target: DataSource, table_conf: Table, database_config: DatabaseConfig,
```
Make this function private
this will help formatter

Reply:
done